android/app/src/main/java/BridgePackage.java

```diff
@@ -0,0 +1,36 @@
+package com.my.package;
+
+
+import com.facebook.react.ReactPackage;
+import com.facebook.react.bridge.JavaScriptModule;
+import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.uimanager.ViewManager;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class BridgePackage implements ReactPackage {
+
+    @Override
```

Comment:
[from RN 0.47.x on](https://github.com/facebook/react-native/releases/tag/v0.47.2) the `createJSModules()` method has been removed and thus the `@Override` implement here will fail the gradle compilation. [Simply remove the `@Override` line and keep the method call](https://github.com/facebook/react-native/commit/ce6fb337a146e6f261f2afb564aa19363774a7a8) should solve this issue.

Reply:
Thanks man!